README.md

```diff
@@ -1,2 +1,29 @@
 # comp110-worksheet-B
-Base repository for COMP110 worksheet B
+
+
+## Pseudocode:
+
+```
```

Comment:
The worksheet asked only for the calculation of the likeness score, not the whole game! Careful reading and implementation of specifications is an important skill for software developers to master.